### PR TITLE
Bump firewall and firewall-multi to latest

### PR DESCRIPTION
**Pull request by ja5087**
It finally landed

### CODE REVIEW

**Comment by cg505** [comment]
letsgo

File: Puppetfile [hunk 1]
```diff
@@ -19,11 +19,7 @@ mod 'puppet-prometheus',                   '6.2.0'
 mod 'puppetlabs-apache',                   '3.1.0'
 mod 'puppetlabs-apt',                      '6.3.0'
 mod 'puppetlabs-concat',                   '4.2.1'
-# Make sure to update alexharvey/firewall_multi version
```
Comment (cg505) [inline Puppetfile:22]: this comment should have been kept @ja5087

Comment (ja5087) [reply Puppetfile:22]: oh yikes

Comment (cg505) [reply Puppetfile:22]: nw

**Pull request merged**